Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM python:3.7-slim
 WORKDIR /benchmark
 COPY . /benchmark
 
+RUN apt-get update && apt-get install -y gcc python3-dev
```

Comment:
I think it would be worth adding a comment to say these dependencies are required for the psutil Python package.